python/cudf/cudf/tests/test_binops.py

```diff
@@ -488,79 +488,83 @@ def test_boolean_scalar_binop(op):
 
-    utils.assert_eq(expect, got)
+    utils.assert_eq(ps_result, cs_result)
 
 
-@pytest.mark.parametrize("func", _operator_funcs + _operator_funcs_series)
+@pytest.mark.parametrize("func", _operators_arithmatic)
 @pytest.mark.parametrize("has_nulls", _nulls)
 @pytest.mark.parametrize("fill_value", [None, 27])
 @pytest.mark.parametrize("dtype", ["float32", "float64"])
 def test_operator_func_series_and_scalar(dtype, func, has_nulls, fill_value):
-    nelem = 1000
-    arr = utils.gen_rand(dtype, nelem) * 10000
-    scalar = 59.0
+    count = 1000
+    scalar = 59
+    cs = utils.gen_rand_series(dtype, count, has_nulls=has_nulls, stride=10000)
+    ps = cs.to_pandas()
 
-    if has_nulls == "some":
-        nulls = utils.random_bitmask(nelem)
-        sr = Series.from_masked_array(arr, nulls)
-    else:
-        sr = Series(arr)
+    cs_result = getattr(cs, func)(scalar, fill_value=fill_value)
+    ps_result = getattr(ps, func)(scalar, fill_value=fill_value)
 
-    psr = sr.to_pandas()
+    utils.assert_eq(ps_result, cs_result)
 
-    expect = getattr(psr, func)(scalar, fill_value=fill_value)
-    got = getattr(sr, func)(scalar, fill_value=fill_value)
 
-    # This is being done because of the various gymnastics required to support
-    # equality for null values. cudf.Series().to_pandas() replaces nulls with
-    # None and so a bool Series becomes object Series. Which does not match the
-    # output of equality op in pandas which remains a bool. Furthermore, NaN
-    # values are treated as not comparable and always return False in a bool op
-    # except in not-equal op where bool(Nan != Nan) gives True.
-    if got.dtype == np.bool:
-        got = got.fillna(True) if func == "ne" else got.fillna(False)
+@pytest.mark.parametrize("dtype", ["float32", "float64"])
+@pytest.mark.parametrize("func", _operators_logical)
+@pytest.mark.parametrize("has_nulls", _nulls)
+@pytest.mark.parametrize("fill_value", [None, True, False, 1.0])
+def test_operator_func_between_series_logical(
+    dtype, func, has_nulls, fill_value
+):
+    count = 1000
+    cs_a = utils.gen_rand_series(
+        dtype, count, has_nulls=has_nulls, stride=10000
+    )
+    cs_b = utils.gen_rand_series(dtype, count, has_nulls=has_nulls, stride=100)
+    ps_a = cs_a.to_pandas()
+    ps_b = cs_b.to_pandas()
 
-    utils.assert_eq(expect, got)
+    cs_result = getattr(cs_a, func)(cs_b, fill_value=fill_value)
+    ps_result = getattr(ps_a, func)(ps_b, fill_value=fill_value)
+
+    utils.assert_eq(ps_result, cs_result)
+
+
+@pytest.mark.parametrize("dtype", ["float32", "float64"])
+@pytest.mark.parametrize("func", _operators_logical)
+@pytest.mark.parametrize("has_nulls", _nulls)
+@pytest.mark.parametrize("scalar", [-59.0, np.nan, 0, 59.0])
+@pytest.mark.parametrize("fill_value", [None, True, False, 1.0])
+def test_operator_func_series_and_scalar_logical(
+    dtype, func, has_nulls, scalar, fill_value
+):
+    cs = utils.gen_rand_series(dtype, 1000, has_nulls=has_nulls, stride=10000)
```

Comment:
Just a minor suggestion: we have a few different conventions for naming Pandas and cuDF objects in tests. Some that come to mind:

1. `pdf/gdf`
2. `pdf/cdf`
3. `df/gdf`
4. `pdf/df`

Suggest that we adopt one of the above and avoid the [standards problem](https://xkcd.com/927/) :)